staging/istio/Chart.yaml

```diff
@@ -1,8 +1,7 @@
 apiVersion: v1
```

Comment:
If we stick with `apiextensions.k8s.io/v1`, this should be switched to v2 as tiller does not support this.

Reply:
```suggestion
apiVersion: v2
```

Reply:
:+1: